### PR TITLE
Gulp serve, watch and transform

### PR DESCRIPTION
**Pull request by dzito**
Runs a local web server, watches for changes to files in any `src/` directory, transforms JSX to JS, and automatically reloads the browser.

Updated the README.md with the two required setup steps. @grantt - what do you think about doing away with the environment_setup.html? It might be easier for newcomers to find setup steps in the README.

### CODE REVIEW

**Comment by dzito** [comment]
Try running `npm install` and `gulp` on this branch. Let me know if any dependencies were not specified correctly, since my system has a bias.

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  browserSync = require('browser-sync'),
+  reload = browserSync.reload;
+  
+var paths = {
+  startUrl: '/lesson_1/lesson.html',
+  scripts: ['lesson_*/src/*.js']
+};
+ 
+gulp.task('server', function() {
+  browserSync({
```
Comment (marsjosephine) [inline gulpfile.js:14]: Out of curiosity, any reason to use  `browser-sync` instead of `gulp-connect` here?

Comment (dzito) [inline gulpfile.js:14]: No idea. What's the difference?

Comment (marsjosephine) [inline gulpfile.js:14]: Both `gulp-connect` and `browser-sync` can run web servers, but `browser-sync` has a little more functionality for developer workflow. Both would work here! Just curious if there was a reason to use one over the other :smile:

**Comment by marsjosephine** [comment]
@dzito -- we're going to open up the class to the rest of engineering and these changes would be great to have! Would you mind merging forward so we can merge this PR into master?

**Comment by dzito** [comment]
sure, but feel free to DIY since I may not get to this today. :)

**Comment by marsjosephine** [comment]
@dzito -- sure haha. Will do!

**Comment by grantt** [comment]
🎉

**Pull request merged**